pages/common/fls.md

```diff
@@ -0,0 +1,16 @@
+# fls
+
+> List files and directories in an image file or device.
+> More information at https://wiki.sleuthkit.org/index.php?title=Fls.
```

Comment:
```suggestion
> More information: <https://wiki.sleuthkit.org/index.php?title=Fls>.
```
It should be in the standard to add a more information, as in the other tldr-pages.